src/dosext/mfs/mfs.c

```diff
@@ -1659,6 +1660,12 @@ dos_fs_dev(struct vm86_regs *state)
 
   Debug0((dbg_fd, "emufs operation: 0x%08x\n", WORD(state->ebx)));
 
+  if (WORD(state->ebx) == DOS_SUBHELPER_MFS_EMUFS_LOADED) {
```

Comment:
How about `DOS_SUBHELPER_MFS_EMUFS_INIT`
that returns success only for the first time and CARRY
otherwise?